k8s-ci/test

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -euo pipefail
+cd $(dirname $0)
+. ./utils.sh
+
+# Set environment variables
+export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
+export SECRETLESS_IMAGE="${DOCKER_REGISTRY_PATH}/secretless-broker:${UNIQUE_TEST_ID}"
+
+# Clean up when script completes
+function finish {
+  announce 'Wrapping up and removing test environment'
+
+  # Delete registry images that were used
+  deleteRegistryImage "${SECRETLESS_IMAGE}" > /dev/null
```

Comment:
also the gke-utils image?

Reply:
nah the gke-utils image is for the CI host. if having it around can speed up subsequent builds that can only be a good thing